codalab/static/teams/teams.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+	$('.remove-image .btn').click(function() {
```

Comment:
Very nice stuff here, thanks! Final thing, could you fix indentation here to 4 space?